tpl/config.ini.gm

```diff
@@ -37,6 +41,21 @@
     ;the ca certificate file
     ca_cert=gmca.crt
 
+; storage security releated configurations
+[storage_security]
+; enable storage_security or not
+;enable=true
+; the IP of key mananger
+;key_manager_ip=
+; the Port of key manager
+;key_manager_port=
+;cipher_data_key=
+
+[chain]
+    id=1
+[compatibility]
+    supported_version=2.0.0-rc1
```

Comment:
this should be newest default

Reply:
Generator cant changed this section ··supported_version·· now